chips/stm32/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "stm32"
+version = "0.1.0"
+authors = ["Tock Project Developers <tock-dev@googlegroups.com>"]
+
+[dependencies]
+kernel = { path = "../../tock/kernel" }
```

Comment:
Could also just use the git repo as a dependency instead of using a submodule. _Might_ be better assuming there isn't a plan to modify the kernel from this repo